gherkin/php/composer.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "cucumber/gherkin",
+    "description": "Gherkin parser",
+    "author": "Cucumber Limited <cukes@googlegroups.com>",
+    "license": "MIT",
+    "type": "library",
+    "autoload": {
+        "psr-4": {
+            "Cucumber\\Gherkin\\": ["src/", "src-generated/"]
+        }
+    },
+    "repositories": [
```

Comment:
This needs to be removed for a release

Reply:
Such thing is done with other implementation in the pre-release make target.
For example with ruby, we comment some lines in the Gemfile: https://github.com/cucumber/common/blob/main/gherkin/ruby/default.mk#L48
Or with Go, we remove some "replace" instructions: https://github.com/cucumber/common/blob/main/gherkin/go/default.mk#L86

Reply:
Great, I've added a target using `jq` to remove this block during `pre-release`